LiteDB/Engine/FileReader/FileReaderV7.cs

```diff
@@ -112,7 +112,7 @@ public IEnumerable<BsonDocument> GetDocuments(string collection)
 
                         if (dataPage["pageType"].AsInt32 != 4) continue;
 
-                        var block = dataPage["blocks"].AsArray.FirstOrDefault(x => x["index"] == dataBlock["index"]).AsDocument;
+                        var block = dataPage["blocks"].AsArray.FirstOrDefault(x => x["index"] == dataBlock["index"])?.AsDocument;
```

Comment:
FirstOrDefault - can return null

Reply:
Not sure if it's actually possible to return "null" here. But in that case, it might be faster to use .First() instead of .FirstOrDefault()

Reply:
next line will be
``` 
if (block == null) continue;
```
so we can return null, and as I checked no big differents betwen First and FirstOrDefault
https://github.com/microsoft/referencesource/blob/master/System.Core/System/Linq/Enumerable.cs
FirstOrDefault will retrn null, and First just throw an exeption